src/components/BnPagination/BnPagination.vue

```diff
@@ -74,7 +81,10 @@ function parseUrl(urlTemplate: string, page: number) {
     :is="props.as"
     class="bn-pagination"
   >
-    <ul class="bn-pagination__list">
+    <ul
+      class="bn-pagination__list"
+      :class="props.classes.list"
+    >
       <li v-if="pageItems.prev">
```

Comment:
Maybe not in this PR but it'd be nice to add extra classes like the `li` even though they are not being used by the default style.

Reply:
I'll add it to the board. To clarify, you mean a BEM class, a class through the `classes` prop, or both?

Reply:
I was thinking about the `classes` prop but now that you mention it, it probably should also be a BEM class.

Reply:
I'll add it